modules/ROOT/pages/to-share-an-asset.adoc

```diff
@@ -0,0 +1,62 @@
+= Share an Asset
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+:page-aliases: to-share-an-asset-with-a-user.adoc
+
+Exchange administrators and asset administrators can share an asset with another user in their organization, with the whole organization, or with an external organization.
+
+Any Exchange user can click the *Shared with me* link in the left navigation menu of Exchange to show all assets shared with their user account and all assets shared with their organization.
+
+== Share With a User
+
+If the person is not in your organization, click *Access Management* > *Users* to add them.
+
+. Click an asset to show its details.
+. Click *Share*.
+. Begin typing the name of the person you want to share this asset with, and select that person in the list. You can optionally click a filter to show only users who have Viewer, Contributor, or Admin rights.
+. Click *Add*.
```

Comment:
We should remove this, there is no add button, you can select the user by clicking in the autocomplete row, or pressing enter (it will add the first result to the list).

Reply:
Thank you, and I'm sorry about that! I walked through the UI to clarify this and removed the Add step.